stacks.yml

```diff
@@ -543,6 +543,17 @@ jenkins_ssh_access:
     ToPort: 22
     SecurityGroup: "{{ stacks.jenkins.outputs.InstanceSecurityGroup }}"
 
+jenkins_letsencrypt_access:
```

Comment:
Letsencrypt creates a standalone server for a few minutes when it updates a certificate, so we do need it open every few months.

I went back and forth on keeping it open all the time. What I've done is move all nginx stuff to 443, so Jenkins and log proxies are only available via HTTPS and the security group for 443 only allows  AH IPs. Which should make it safe to open port 80 for everyone, so that we can use it for renewing letsencrypt certs. We could close it when we don't need it to avoid accidental nginx misconfiguration, but apart from that it shouldn't make any difference since nothing is listening most of the time.